configs/getConfig.js

```diff
@@ -0,0 +1,20 @@
+const _ = require("lodash");
```

Comment:
@Pomax I agree. I've advocated for limiting our use of lodash. I would suggest removing it from this file as well.